.github/workflows/docker.yml

```
@@ -6,37 +6,48 @@ on:
     branches:
     - develop
 
+
+permissions:
+  contents: read
+
 jobs:
   build_docker_image:
     # This workflow is only of value to PyBaMM and would always be skipped in forks
     if: github.repository_owner == 'pybamm-team'
     name: Build image
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
```

```suggestion
    permissions:
      contents: read
```

We aren't working with GitHub Packages, are we? We push only to Docker Hub, and not the GH Container Registry. Please correct me if I'm wrong!